FormalConjectures/ErdosProblems/730.lean

```diff
@@ -19,21 +19,22 @@ import FormalConjectures.Util.ProblemImports
 /-!
 # Erdős Problem 730
 
-*Reference:* [erdosproblems.com/730](https://www.erdosproblems.com/730)
+*References:*
+  - [erdosproblems.com/730](https://www.erdosproblems.com/730)
+  - [A129515](https://oeis.org/A129515)
 -/
 private abbrev S :=
-  {(n, m) : ℕ × ℕ | n ≠ m ∧ ((2*n).choose n).primeFactors = ((2*m).choose m).primeFactors}
+  {(n, m) : ℕ × ℕ | n < m ∧ ((2*n).choose n).primeFactors = ((2*m).choose m).primeFactors}
```

Comment:
How come this change? On the website it still says `≠`

Reply:
Ah, I guess it doesn't matter if you are asking for the infinitude of this set, so I don't mind if it makes things simpler.